api/jobs.go

```diff
@@ -151,6 +144,12 @@ func (api *API) GetJobsHandler(w http.ResponseWriter, req *http.Request) {
 	}
 }
 
+func (api *API) setUpPagination(w http.ResponseWriter, offsetParam string, limitParam string, ctx context.Context) (int, int, error) {
+	paginator := pagination.NewPaginator(api.cfg.DefaultLimit, api.cfg.DefaultOffset, api.cfg.DefaultMaxLimit)
+	offset, limit, err := paginator.ValidatePaginationParameters(offsetParam, limitParam)
+	return offset, limit, err
```

Comment:
No need to store the variables?

```suggestion
	return paginator.ValidatePaginationParameters(offsetParam, limitParam)
```

Reply:
no I guess not, good point - removed